src/codelabs/async-await.md

```diff
@@ -42,6 +42,17 @@ another operation to finish. Here are some common asynchronous operations:
 To perform asynchronous operations in Dart, you can use the `Future` class
 and the `async` and `await` keywords.
 
+Typically, APIs that return `Future`s fall into one of the following categories:
```

Comment:
Instead of [making a code element plural](https://developers.google.com/style/code-in-text?hl=en#grammatical-treatment-of-code-elements), we add a noun:

```suggestion
Typically, APIs that return `Future` objects
fall into one of the following categories:
```